base/ubi9/entrypoint.sh

```diff
@@ -5,6 +5,18 @@ if [ ! -d "${HOME}" ]; then
   mkdir -p "${HOME}"
 fi
 
+# Configure container builds to use vfs or fuse-overlayfs
+if [ ! -d "${HOME}/.config/containers" ]
```

Comment:
@cgruver a matter of preference, but maybe you can change the formatting to follow the style applied in this file e.g. single line `if []; then`

Reply:
Sure.  Will do.

Reply:
I adjusted the formatting to match.